### PR TITLE
Fix typos in zkGenConfig.sh logs

### PR DESCRIPTION
**Pull request by guillaumebreton**
This patch fix two typos in the zkGenConfig.sh script logs

### CODE REVIEW

File: statefulsets/zookeeper/zkGenConfig.sh [hunk 1]
```diff
@@ -80,7 +80,7 @@ function validate_env() {
     echo "ZK_PURGE_INTERVAL=$ZK_PURGE_INTERVAL"
     echo "ENSEMBLE"
     print_servers
-    echo "Enviorment validation successful"
+    echo "Enviroment validation successful"
```
Comment (enisoc) [inline statefulsets/zookeeper/zkGenConfig.sh]: Still missing an 'n'.

**Comment by guillaumebreton** [comment]
@enisoc I fixed it sorry. I also remove useless spaces at the end of lines.

**Comment by enisoc** [comment]
@guillaumebreton Looks good now. Can you just rebase onto latest master?

/assign

**Pull request merged**